File: test/fixtures/hello.c
#include <stdio.h>

int main(int argc, char** argv) {
  FILE *file;
  char c;

  printf("Hello EXE World");
  if (argc > 1) {
    printf(", arguments passed\n");
    file = fopen(argv[1], "r");
    if (file) {
      while ((c = fgetc(file)) != EOF) {
        printf("%c", c);
      }
      fclose(file);
    } else {
      printf("ERROR: could not open file");
    }
  }
  printf("\n");
  return 0;
}

File: test/fixtures/hello.cs
using System;

public class HelloWorld
{
  public static void Main(string[] args)
  {
    Console.Write("Hello DotNet World");
    if (args.Length > 0) {
      Console.WriteLine(", arguments passed");
      string filename = args[0];
      if (System.IO.File.Exists(filename)) {
        Console.Write(System.IO.File.ReadAllText(args[0]));
      } else {
        Console.Write("ERROR: could not open file");
      }
    }
    Console.WriteLine("");
  }
}
